koku/masu/processor/aws/aws_report_parquet_summary_updater.py

```diff
@@ -57,9 +57,10 @@ def update_summary_tables(self, start_date, end_date, **kwargs):
 
         """
         start_date, end_date = self._get_sql_inputs(start_date, end_date)
+        ec2_compute_summary_table = AWS_CUR_TABLE_MAP["ec2_compute_summary"]
 
         with schema_context(self._schema):
-            partition_summary_tables = UI_SUMMARY_TABLES + (AWS_CUR_TABLE_MAP["ec2_compute_summary"],)
+            partition_summary_tables = UI_SUMMARY_TABLES + (ec2_compute_summary_table,)
```

Comment:
This could also be done with unpacking making it read a little better and removing potential maintenance errors with single item tuples.
```suggestion
            partition_summary_tables = (*UI_SUMMARY_TABLES, ec2_compute_summary_table)
```

Reply:
I've update to more readable format, thanks for the suggestion